src/Calendar/MonthDropdown.tsx

```diff
@@ -76,7 +80,7 @@ class MonthDropdown extends React.PureComponent<MonthDropdownProps> {
     const { disabledMonth } = this.props;
 
     if (disabledMonth) {
-      const days = getDaysInMonth(new Date(year, month));
+      const days = getDaysInMonth(legacyParse(new Date(year, month)));
```

Comment:
legacyParse is not required when Date type create by `new`
see: https://github.com/date-fns/date-fns-upgrade/blob/bd0cfffeec77ef1a4efcf4e9714e42dbb1c7e842/src/v2/legacyParse/index.ts#L52

Reply:
Must be used in the local week because of this issue https://github.com/rsuite/rsuite/issues/892
If you add the props "IsoWeek", the weeks are correct https://www.epochconverter.com/weeks/2020 / http://myweb.ecu.edu/mccartyr/isowdcal.html

See attachments:
# Normal
<img width="595" alt="Screen Shot 2020-06-25 at 11 51 50" src="https://user-images.githubusercontent.com/26462869/85743447-6ad9b600-b6da-11ea-9b68-cd1d2ae46fc1.png">
# IsoWeek
<img width="597" alt="Screen Shot 2020-06-25 at 11 52 00" src="https://user-images.githubusercontent.com/26462869/85743463-6d3c1000-b6da-11ea-9b11-3197c6a355bb.png">